plugins/copybuffer.zsh

```diff
@@ -0,0 +1,14 @@
+# copy the active line from the command line buffer 
+# onto the system clipboard (requires clipcopy plugin)
+
+copybuffer () {
+  if which clipcopy &>/dev/null; then
```

Comment:
[`clipcopy` is a function provided by OMZ](https://github.com/robbyrussell/oh-my-zsh/blob/d51ee38/lib/clipboard.zsh#L18). This check shouldn't be needed but it's ok.